Detectors/TPC/workflow/src/RecoWorkflow.cxx

```diff
@@ -209,6 +212,7 @@ framework::WorkflowSpec getWorkflow(CompletionPolicyData* policyData, std::vecto
     // PublisherSpec with only sector 0, '_0' is thus appended to the branch name
     specs.emplace_back(o2::tpc::getPublisherSpec(PublisherConf{
                                                    "tpc-compressed-cluster-reader",
+                                                   "tpc-compressed-cluster.root",
                                                    "tpcrec",
                                                    {"clusterbranch", "TPCCompClusters", "Branch with TPC compressed clusters"},
                                                    {"clustermcbranch", "TPCClusterNativeMCTruth", "MC label branch"},
```

Comment:
When we are cleaning up anyway, perhaps wee should remove the "clustermcbranch" branch from the publisher for compressed clusters, since it is not present. No idea if we can instantiate the processor without this line, but perhaps send empty strings or whatever specific name to indicate this is unsupported behavior.

Reply:
Some {} initialization is needed, unless we introduce explicit c-tors for PublisherConf, not sure it is worth to complicate it.
Passing just an empty string works,